### PR TITLE
Inline bidding experience

### LINKED ISSUE #701: Inline bid experience for eligible vendor
As a vendor who is eligible to bid on an auction, I want to easily be able to do this and see if my bid was accepted or why it was not accepted. 

Context:
I am an eligible vendor
And there is a live auction that I'm eligible for
And I have entered my bid in the "Your Bid" box
When I click on the "Place Bid" button

Scenario: Eligible vendor submits bid - no errors
And there are no errors
Then I should be taken to the existing confirmation page

Scenario: Vendor is redirected to show page for the auction after confirming the bid
When I click on the confirm button
Then my bid should be saved
And I should go back to the show page for that auction
And I should see a success message ("Congratulations, your bid of $XXXX was accepted")

Scenario: Eligible vendor submits bid - with errors
And there are errors
Then my bid should not be recorded
And I should stay on the show page for that auction
And I should see an inline message describing the error (use existing error messaging)

Note: Use existing confirmation page. Just redirect to the auction show page and show success message there

### PR DESCRIPTION
**Pull request by jessieay**
* Closes https://github.com/18F/micropurchase/issues/698
* Closes #701
* Still need to add "You have the lowest bid!" for winner

screen shot 2016-06-21 at 4 57 09 pm

### CODE REVIEW

File: app/views/bids/_form.html.erb [hunk 1]
```diff
@@ -1,30 +0,0 @@
...
-    </div>
-
-    <div class="usa-width-one-half">
-      <h2 class="center"><label for="bid_amount">Your bid:</label></h2>
-      <%= f.text_field :amount %>
-    </div>
-  </div>
-
-  <p>
-    Note that all users who submit bids consent to the publication of their GitHub
```
Comment (jessieay) [inline app/views/bids/_form.html.erb]: do we need to include this warning elsewhere now that we are doing inline bidding? #lawyersneeded cc @adelevie

Comment (harrisj) [inline app/views/bids/_form.html.erb]: I assume this is part of the Terms of Service already, right?

File: app/view_models/auction_show_view_model.rb [hunk 2]
```diff
@@ -66,9 +70,17 @@ def eligibility_label
     end
   end
 
+  def bid_form_prompt
```
Comment (harrisj) [inline app/view_models/auction_show_view_model.rb]: Is this something we should delegate to Rules objects? Or is that polymorphism overkill?

File: features/vendor_views_small_biz_auctions.feature [hunk 3]
```diff
@@ -8,13 +8,13 @@ Feature: Vendor views auctions set aside for small businesses only
     And I am a user with a verified SAM account who is not a small business
     And I sign in
     When I visit the auction page
-    Then I should not see the bid button
+    Then I should not see the bid form
```
Comment (harrisj) [inline features/vendor_views_small_biz_auctions.feature]: :clap:

File: app/view_models/auction_show_view_model.rb (hunk 2 above)
Comment (adelevie) [inline app/view_models/auction_show_view_model.rb]: > Is this something we should delegate to Rules objects?

I think so.

File: app/views/bids/_form.html.erb (hunk 1 above)
Comment (adelevie) [inline app/views/bids/_form.html.erb]: https://github.com/18F/micropurchase/blob/develop/app/views/logins/index.html.erb has it:

> None of the content in this GitHub repository is considered proprietary or procurement-sensitive information. Furthermore, all users who submit bids consent to the publication of their GitHub usernames, DUNS numbers, and bid amounts following the conclusion of the bidding process. Bidders' email addresses will not be disclosed and will only be used for micro-purchase-related communication on an as-needed basis.

Although, looks like we should get some clarification from @allalala about this line: "None of the content in this GitHub repository is considered proprietary". We might want to change that to say "website" or something else.

**Comment by adelevie** [comment]
LGTM ... your call on sweeping `bid_form_prompt` into a rules object.

So cool:

screen shot 2016-06-21 at 9 27 21 pm

**Comment by harrisj** [comment]
Can I say how gorgeous things are looking? Merging.

**Pull request merged**